kernel/packages/shared/apis/RestrictedActionModule.ts

```diff
@@ -0,0 +1,62 @@
+import { exposeMethod, registerAPI } from 'decentraland-rpc/lib/host'
+import { ExposableAPI } from './ExposableAPI'
+import defaultLogger from '../logger'
+import { unityInterface } from '../../unity-interface/UnityInterface'
+import { ParcelIdentity } from './ParcelIdentity'
+import { Vector3 } from '../../decentraland-ecs/src/decentraland/math'
+import { gridToWorld, isInParcel, parseParcelPosition } from '../../atomicHelpers/parcelScenePositions'
+
+enum Permission {
+  ALLOW_MOVE_INSIDE_SCENE = 'ALLOW_MOVE_INSIDE_SCENE'
```

Comment:
I would also change this from `ALLOW_MOVE_INSIDE_SCENE` to `ALLOW_TO_MOVE_PLAYER_INSIDE_SCENE`, to make it consistent with the other renaming